get/application.go

```diff
@@ -200,55 +225,96 @@ func (cmd *applicationsCmd) printStats(ctx context.Context, c *api.Client, appLi
 		return err
 	}
 	w := tabwriter.NewWriter(out, 0, 0, 3, ' ', 0)
-	get.writeHeader(w, "NAME", "REPLICA", "STATUS", "CPU", "CPU%", "MEMORY", "MEMORY%", "RESTARTS", "LASTEXITCODE")
+	get.writeHeader(w, "NAME", "TYPE", "REPLICA", "STATUS", "CPU", "CPU%", "MEMORY", "MEMORY%", "RESTARTS", "LASTEXITCODE")
```

Comment:
Isn't the type already visible from the replica name? Just not sure if we should really make the stats output even wider since it's already super wide if you have a long app name.